.github/workflows/conformance-report.yml

```diff
@@ -96,11 +98,31 @@ jobs:
         run: |
           mkdir -p $GITHUB_WORKSPACE/artifact
           cp -r $GITHUB_WORKSPACE/${{ github.event.pull_request.base.sha }}/$PATH_TO_TEST_RUNNER/$CONFORMANCE_REPORT_RELATIVE_PATH $GITHUB_WORKSPACE/artifact/$CONFORMANCE_REPORT_RELATIVE_PATH
-      # Run conformance report comparison. Generates `comparison_report.md`
-      - name: Run conformance report comparison for artifact. Generates `comparison_report.md`
+      - name: "Artifact CROSS-ENGINE Report (creates comparison_report.md)"
         continue-on-error: true
         run: |
-          ARGS="$GITHUB_WORKSPACE/artifact $CONFORMANCE_REPORT_RELATIVE_PATH ${{ github.event.pull_request.base.sha }} $GITHUB_SHA $COMPARISON_REPORT_NAME"
+          T="CROSS-ENGINE-REPORT"
+          BF="$GITHUB_WORKSPACE/$PATH_TO_TEST_RUNNER/src/main/resources/config/legacy/$CONFORMANCE_REPORT_NAME"
+          BL="LEGACY"
+          BT="$LEGACY_VERSION"
+          TF="$GITHUB_WORKSPACE/$PATH_TO_TEST_RUNNER/$CONFORMANCE_REPORT_RELATIVE_PATH/eval/$CONFORMANCE_REPORT_NAME"
+          TL="EVAL"
+          TT="$GITHUB_SHA"
+          O="$COMPARISON_REPORT_NAME"
+          ARGS="-t $T -bf $BF -bl $BL -bt $BT -tf $TF -tl $TL -tt $TT -o $O"
+          gradle :test:partiql-tests-runner:run --args="$ARGS"
+      - name: "Artifact CROSS-COMMIT Report (appends to comparison_report.md)"
+        continue-on-error: true
+        run: |
+          T="CROSS-COMMIT-REPORT"
+          BF="$GITHUB_WORKSPACE/artifact/eval/$CONFORMANCE_REPORT_NAME"
+          BL="EVAL"
+          BT="${{ github.event.pull_request.base.sha }}"
+          TF="$GITHUB_WORKSPACE/$PATH_TO_TEST_RUNNER/$CONFORMANCE_REPORT_RELATIVE_PATH/eval/$CONFORMANCE_REPORT_NAME"
+          TL="EVAL"
+          TT="$GITHUB_SHA"
+          O="$COMPARISON_REPORT_NAME"
+          ARGS="-t $T -bf $BF -bl $BL -bt $BT -tf $TF -tl $TL -tt $TT -o $O"
```

Comment:
nit: consider using the long names here so we don't have to reference the runner code to understand this.